rust-connectors/sinks/sql/src/db.rs

```diff
@@ -0,0 +1,482 @@
+use anyhow::anyhow;
+use async_trait::async_trait;
+use fluvio_future::tracing::{debug, error};
+use fluvio_model_sql::{Operation, Type, Value};
+use itertools::Itertools;
+use rust_decimal::Decimal;
+use sqlx::any::{AnyConnectOptions, AnyKind};
+use sqlx::database::HasArguments;
+use sqlx::postgres::PgArguments;
+use sqlx::query::Query;
+use sqlx::sqlite::SqliteArguments;
+use sqlx::{Connection, Database, Executor, PgConnection, Postgres, Sqlite, SqliteConnection};
+use std::str::FromStr;
+
+const NAIVE_DATE_TIME_FORMAT: &str = "%Y-%m-%d %H:%M:%S%.f";
+
+pub enum Db {
+    Postgres(Box<PgConnection>),
```

Comment:
The crate we use ( `sqlx`) supports many databases (postgres, mssql, MySQL, sqlite) out of the box. Ideally, we wouldn't need any specialization in our code, it will be done in this crate. But is not fully implemented there yet, that's why we need it on our side, but it should be gone in the future.